bin/insert-mock-hosts

```diff
@@ -190,9 +195,10 @@ os.environ['PGUSER'] = args.db_user or default_user
 os.environ['PGPASSWORD'] = args.db_password or default_password
 
 psql = None
-if not args.output_sql:
-    psql = subprocess.Popen('psql', stdin=subprocess.PIPE, shell=True, text=True)
-    output = psql.stdin
+# Causing errors when trying to use script
+# if not args.output_sql:
+#     psql = subprocess.Popen('psql', stdin=subprocess.PIPE, shell=True, text=True)
+#     output = psql.stdin
```

Comment:
Looked into this, before Python 3.7, the parameter was `universal_newlines` instead of `text`. It also states that `encoding` can be used to trigger opening in text mode, so let's do that:

```suggestion
if not args.output_sql:
    psql = subprocess.Popen('psql', stdin=subprocess.PIPE, shell=True, encoding='utf-8')
    output = psql.stdin
```